spec/models/report_spec.rb

```diff
@@ -176,4 +176,26 @@
       expect(report.reportable_activities).not_to include(cancelled_project)
     end
   end
+
+  describe "#editable?" do
+    all_report_states = Report.states.keys
+    editable_states = Report::EDITABLE_STATES
+    readonly_states = all_report_states - editable_states
+
+    editable_states.each do |state|
+      it "is false when the report is #{state}", state: state do |example|
```

Comment:
Nice! 👍

Reply:
Yeah, I didn't want for future state changes to get missed!